dev/ProgressBar/TestUI/ProgressBarReTemplatePage.xaml

```diff
@@ -400,7 +400,7 @@
             <TextBlock AutomationProperties.Name="ShowErrorText" Text="{x:Bind ShowErrorCheckBox.IsChecked, Mode=OneWay}"/>
 
             <TextBlock Text="Is indeterminate:" Margin="0, 20, 0, 0" FontSize="20"/>
-            <TextBlock AutomationProperties.Name="ShowIsIndeterminateText" Text="{x:Bind ShowIsIndeterminateCheckBox.IsChecked, Mode=OneWay}"/>
+            <TextBlock AutomationProperties.Name="ShowIsIndeterminateText" Text="{x:Bind ShowIsIndeterminateCheckBox_ReTemplate.IsChecked, Mode=OneWay}"/>
```

Comment:
>ShowIsIndeterminateText [](start = 50, length = 23)

I'm confused how changes to this ProgressBarPage fixed the Progress Ring tests..

Reply:
Turns out the same name `ShowIsIndeterminateCheckBox` on `ProgressBarPage.xaml` and `ProgressBarReTemplatePage.xaml` was confusing the tests? It was simply not toggling the checkbox through test methods.

Reply:
But why are the _ring_ tests on a _bar_ page?

Reply:
You're right 🤯 Let me investigate.